README.md

```diff
@@ -1,13 +1,27 @@
-[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/comit-network/community)
+<a href="https://comit.network">
+<img src="logo.svg" height="120px">
+</a>
 
-# COMIT Javascript SDK
+---
 
-A software development kit for developing Javascript applications on top of the COMIT protocol reference implementation [comit-rs](https://github.com/comit-network/comit-rs).
+[COMIT](https://comit.network) is an open protocol facilitating cross-blockchain applications.
+With [COMIT](https://comit.network) you can for examples exchange Bitcoin for any Erc20 token directly with another person.
 
-Currently [comit-rs](https://github.com/comit-network/comit-rs) is shipped as a binary called [cnd](https://github.com/comit-network/comit-rs/releases) (COMIT Network Daemon).
+This repository contains the Javascript software development kit (SDK) that wraps the communication with the [comit-network daemon](https://github.com/comit-network/comit-rs) for Javascript developers.
 
+If you wish to do an atomic swap on your machine or to integrate COMIT into an application (e.g. a DEX) please take a look at the [Getting Started section](https://comit.network/docs/getting-started/create-comit-app/) of the COMIT documentation.
 If you have any question please [reach out to the team in our Gitter chat](https://gitter.im/comit-network/community)!
 
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/comit-network/community)
+
+# COMIT Javascript SDK
+
+The COMIT Javascript SDK enables the integration of the [comit-network daemon](https://github.com/comit-network/comit-rs) into Javascript applications.
+You'll find the [latest comit-sdk version on npmjs](https://www.npmjs.com/package/comit-sdk).
+
+Below you find a short introduction into the SDK. For more detailed information please refer to the [API-documentation](TODO-add-link) in our documentation.
```

Comment:
Don't forget to track this TODO somewhere else. It's easy to lose track of this.

Reply:
Might be better to remove it until we have the final links.